docs/features/init.md

```
@@ -0,0 +1,23 @@
+```bash
+git shiplog init
+```
+
+## Behavior
+- Requires running inside a Git repo with at least one commit.
+- Adds fetch/push refspecs for `refs/_shiplog/*` to enable remote synchronization.
+- Enables `core.logAllRefUpdates` to maintain comprehensive reflog history.
+- Conditionally adds a HEAD push refspec only if no push configuration exists (preserves existing push behavior).
+- Produces a human-friendly confirmation in interactive shells and a plain message when `--boring` is supplied.
+
+## Related Code
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)